bundles/FileExplorerBundle/src/Controller/Admin/AdminController.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Pimcore\Bundle\FileExplorerBundle\Controller\Admin;
+
+use Pimcore\File;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/")
```

Comment:
```suggestion
 * @Route("/admin")
```
Also change the route names.

Reply:
The full route then looks something like /admin/bundle/file-explorer/admin/tree
Do we really need to call it admin controller? Can't we just rename it to SettingsController?

Reply:
Good point. I also vote for `SettingsController` and move it out of `/Admin` folder